EIPS/eip-draft_grant_standard.md

```diff
@@ -9,50 +9,50 @@ category: ERC
 created: 2019-04-30
 ---
 
-## Simple Summary
```

Comment:
We should probably leave this in? Seems like it's something expected for each EIP? ("abstract" is a 200 words summary, "simple summary" should just be sentence)

Reply:
Idk. It felt redundant. I was looking at the ERC-20 proposal from way back, and he took it out

https://github.com/ethereum/eips/issues/20

Reply:
We can do that. Idk. It felt redundant